R/teal_slice.R

```diff
@@ -483,44 +530,51 @@ c.teal_slices <- function(...) {
   )
 }
 
+# store method for `teal_slices` object
+#' @param x `teal_slices` object
+#' @param file a `character(1)` with the storage destination name
+#' @export
+#' @rdname teal_slice
+#' @keywords internal
+#'
+store.teal_slices <- function(x, file = NULL, ...) {
+  checkmate::assert_character(file, len = 1, null.ok = TRUE)
```

Comment:
```suggestion
  checkmate::assert_class(x, "teal_slices")
  checkmate::assert_path_for_output(file, overwrite = TRUE, extension = "json")
```
extension argument up for discussion

Reply:
@chlebowa but it's already and overloaded S3 generic method `store.teal_slices` so there is no need for `assert_class`. However I will make it a regular function called `store_filters` and then we should use this assertion

Reply:
https://github.com/insightsengineering/teal.slice/pull/339/commits/3b5e95db795709f114efee6d51c0d9b021938383